mysql/infoschema.go

```diff
@@ -304,6 +310,120 @@ func getConstraints(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]str
 	return primaryKeys, m, nil
 }
 
+// getForeignKeys return list all the foreign keys constraints.
```

Comment:
This comment really belongs on processColumns, but github won't let me put a comment there, so adding it here.

Now that we handle foreign keys here, we should stop handling them in processColumns.
Suggest the following cleanup: replace the for loop that iterates over the constraints with

		for _, c := range constraints[colName] {
			// c can be UNIQUE, PRIMARY KEY, FOREIGN KEY or CHECK
			switch c {
			case "UNIQUE":
				unique = true
			case "CHECK":
				ignored.Check = true
			case "FOREIGN KEY", "PRIMARY KEY"
				// Nothing to do here -- these are both handled elsewhere.
			}
		}

Reply:
updated FOREIGN KEY and PRIMARY KEY case from both postgres and mysql, to not ignore anymore, refactored tests accordingly

Reply:
Thanks!